crates/ironrdp-vmconnect/Cargo.toml

```
@@ -0,0 +1,33 @@
+[package]
+name = "ironrdp-vmconnect"
+version = "0.1.0"
+edition.workspace = true
+license.workspace = true
+homepage.workspace = true
+repository.workspace = true
+authors.workspace = true
+keywords.workspace = true
+categories.workspace = true
```

issue: Disable publish for now, otherwise release-plz will try to publish immediately.

```suggestion
categories.workspace = true
publish = false
```